catroid/src/main/java/org/catrobat/catroid/content/bricks/GoToBrick.java

```diff
@@ -305,7 +306,9 @@ public boolean onTouch(View paramView, MotionEvent paramMotionEvent) {
 
 	@Override
 	public Brick clone() {
-		return new GoToBrick(destinationSprite);
+		GoToBrick copy = new GoToBrick(destinationSprite);
+		copy.spinnerSelection = spinnerSelection;
+		return copy;
```

Comment:
Great idea!
The `setSpinnerSelection` method is called from methods where there is a context. Instead of getting the context from the `CatroidApplication`, you could pass in a parameter context instead.